src/Organizations/OrganizationVersion/components/OrganizationContactInfoVersionView/components/ContactCardHeaderVersionView/ContactCardHeaderVersionView.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import { useContext } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { VersionViewContext } from '@folio/stripes-acq-components';
+
+import css from './ContactCardHeaderVersionView.css';
+
+export const ContactCardHeaderVersionView = ({ isPrimary, name }) => {
+  const versionContext = useContext(VersionViewContext);
+
+  const isUpdated = versionContext?.paths?.includes(name);
+
+  const headerStart = (
+    <h4 className={css.cardHeader}>
+      <span className={isUpdated ? css.mark : ''}><FormattedMessage id={`ui-organizations.${isPrimary ? 'primaryItem' : 'alternateItem'}`} /></span>
```

Comment:
can be set into a new line
```suggestion
      <span className={isUpdated ? css.mark : ''}>
         <FormattedMessage id={`ui-organizations.${isPrimary ? 'primaryItem' : 'alternateItem'}`} />
       </span>
```